readme.md

```diff
@@ -118,6 +118,11 @@ GitHub Enterprise is also supported: [How to enable it](https://fregante.github.
 		<th width="50%">
 			<p><a title="select-notifications"></a> Select notifications by type and status
 			<p><img src="https://user-images.githubusercontent.com/1402241/152119039-4ea8333f-a744-4106-b56f-cf09f50678be.gif">
+	<tr>
+		<th width="50%">
+			<p><a title="next-prev-notifications"></a> Adds Next Previous Buttons to UI 
+			<p><img src="https://user-images.githubusercontent.com/34628529/229335604-d36e4545-ad88-449d-94c4-a04535796577.png">
```

Comment:
This is not a highlight, please move it to the right section (look for other notification-related features)